airflow/sensors/sql.py

```diff
@@ -81,6 +81,7 @@ def _get_hook(self):
             'odbc',
             'oracle',
             'postgres',
+            'trino',
```

Comment:
I think it's minor, but so far the list is A-Z... why not stick with the convention?

Reply:
Updated. Anywhere else we should keep it in order?